### DISCUSSION TITLE
Temp default to C

### DISCUSSION THREAD

**Discussion by aidzer0** [aldostools/webMAN-MOD#729] [Q&A] [answered]
Good day, I have been trying to figure out how to set the FPS counter temp to show Celsius only, could anyone please point me in the right direction?

Thank you!

**Comment by aldostools** [comment, marked as answer]
Go to http://0/edit.ps3/dev_hdd0/tmp/wm_res/VshFpsCounter.yaml in PS3 browser

Change the line at the bottom (line 13) to:
````yaml
temperatureType: CELSIUS # Available options BOTH, CELSIUS, FAHRENHEIT 
````

Save, and load FPS counter with `L3`+`R3`

**Reply by aidzer0** [reply]
EPIC, thank you!!

**Comment by 760194962** [comment]
Thanks for explanation.
I notice that in psx game mode the fps displayed in top left still switch between C and F, anyway it may be modified? I have changed both line in yaml file you said above. "xmb" and "game"

**Reply by aldostools** [reply]
Check if you saved the changes in `VshFpsCounter.yaml`. If you did, you have to reload VshFpsCounter again.

If the issue persists, you could check previous verions in the following link or report the bug to the author of the plugin:
https://github.com/TheRouletteBoi/RouLetteVshMenu/releases
